### ISSUE TITLE
[7.0] Searching with Nested Relationships

### ISSUE THREAD

**Issue by stsepelin** [yajra/laravel-datatables#789]
### Summary of problem or feature request

Hello,

I'm trying to build table with posts and users, each user have moderator. So table contains $post->id, $post->title, $post->user->email and $post->user->moderator->email. There is no problem with displaying content. But I can't understand how to use search with such type of relationships ($post->user->moderator->email).

This is part of my js:

```
columns: [
                    { data: 'id', name: 'id' },
                    { data: 'title', name: 'title' },
                    { data: 'user', name: 'user.email' },
                    { data: 'moderator', name: 'moderator' },
                ]
```

But now, if I try to search, I'm getting error that "Column not found".
I also tried to do something like this:

```
columns: [
                    { data: 'id', name: 'id' },
                    { data: 'title', name: 'title' },
                    { data: 'user', name: 'user.email' },
                    { data: 'moderator', name: 'user.moderator.email' },
                ]
```

But it doesn't work.

I'm using last version of yajra/laravel-datatables and laravel 5.3

**Comment by yajra** [comment]
@stsepelin sorry but only first level of relationship is currently tested to work with the package. Will try to implement this when I had the chance. Thanks!

**Comment by yajra** [comment]
Should be fixed on v6.22.0. Thanks!
